### DISCUSSION TITLE
DNSSEC not available

### DISCUSSION THREAD

**Discussion by paulposition** [poweradmin/poweradmin#432] [Q&A]
Hi all
I'm very happy to see that this project is still alive!

on a brand new installation, almalinux 8.5, pdns 4.6, php 8.1, I cannot see DNSSEC checkbox in zone list page.
I can see when adding a new domain, but not in the zone list page, nor any related record in the zone...
can someone help me pls?
thanks rgds
Paolo

**Comment by edmondas** [comment]
@paulposition please check the latest release. If the issue persists please let me know.

**Reply by edmondas** [reply]
Also do not forget to set `$pdnssec_use = true;` in your config file (config.inc.php).

**Comment by rmwerder** [comment]
After putting $pdnssec_use=true into config.inc.php I also had to make sure that the webserver user could access /etc/powerdns/pdns.conf AND /etc/powerdns/pdns.d/*

Then suddenly not only the "lock" symbols showed up at the overview, but also the buttons to sign, unsign snd DNSSEC (that is, look at the data)

**Reply by edmondas** [reply]
That's right, depending on your system settings, calling pdnsutil from the web server may require rights to access pdns config. Another option is the use of DNSSEC via the PowerDNS API.

```
$pdnssec_use = true;
$pdns_api_url = 'URL'; // Replace the PowerDNS API URL with your own
$pdns_api_key = 'KEY'; // Enter your API key here.
```
